scripts/launch_aws_cluster.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+
+# COPS-4642: using fork of terraform-aws-instance as spot instances are unsupported in dcos-terraform.
+dcos_terraform_instance_aws="git::https://github.com/dcos/terraform-aws-spot-instance.git?ref=${DCOS_TERRAFORM_VERSION:-0.2.0}"
+
+# Forked terraform:
+
+# Terraformfile support to allow overriding module sources: https://github.com/hashicorp/terraform/pull/20229
+URL_terraform="https://downloads.mesosphere.com/dcos-terraform/terraform/v0.11.13-mesosphere/${OS}_amd64/terraform"
+
+# Forked terraform-provider-aws:
+
+# Workaround IAM role creation race with spot instances: https://github.com/terraform-providers/terraform-provider-aws/pull/8556
+# Retry InvalidTarget errors for AWS LB target group attachment: https://github.com/terraform-providers/terraform-provider-aws/pull/8538
+# Make the ready timeout configurable for ELB and spot instance requests: https://github.com/terraform-providers/terraform-provider-aws/pull/8180
+URL_terraform_provider_aws="https://downloads.mesosphere.com/dcos-terraform/terraform/v0.11.13-mesosphere/${OS}_amd64/terraform-provider-aws"
+
+if [ "$OS" == "linux" ]; then
+    CHECKSUM_terraform="19b19b9c979a75b523ce8ad044bade7a0987ce38f866cfe5479068531839f70c"
+    CHECKSUM_terraform_provider_aws="245ba160a0389c2177091e2dbc72d79d034f1b0a93a5fee01d0d4b69f22be4e1"
+elif [ "$OS" == "darwin" ]; then
+    CHECKSUM_terraform="9bf9567bb0d396f83dd8ee2cedde5f188bdbba3316528648798e1d3f6814a1f7"
+    CHECKSUM_terraform_provider_aws="bdb07595b393c2fba7e0a679e1d9355e9b30edeec138f48fd1ff9deb2065a8f9"
+else
+    # How did you even get here?
```

Comment:
😄

Reply:
Take a look at https://github.com/dcos/dcos-core-cli/pull/387 it's the first step to unify windows Mac and Linux integration tests.